### ISSUE TITLE
Shoreditch: Make default font color darker

### ISSUE THREAD

**Issue by KokkieH** [Automattic/themes#1768]
We have a user request to make the default font color darker.

The current default font color is #7a7c84 and the user claims the contrast ratio of text on background is 4.16 for that color, while an acceptable ratio is 4.5

Trying to help them with CSS instead.

* User report: https://wordpress.org/support/topic/default-font-color/

**Comment by nielslange** [comment]
@KokkieH I made the default colour slightly darker so that it passed the 4.5 ratio. You can find my PR via https://github.com/Automattic/themes/pull/1769.

**Comment by KokkieH** [comment]
Nice, thanks, @nielslange !

**Comment by kathrynwp** [comment]
We don't normally change colours on themes released so long ago, as it affects all users of the theme, some of whom may not want the surprise change. 

As this user can work around their issue with CSS, I don't think this should be merged.

**Comment by nielslange** [comment]
@kathrynwp While I understand your concerns, I don't think that many people will notice the change given the fact that it's so minimal. Even when viewing the before and after screenshots on https://github.com/Automattic/themes/pull/1769 next to each other, it's hard to spot the difference. That said, I don't have the authority to merge this change and only wanted to share my point of view.

**Comment by ukAndrzej** [comment]
I agree this suggestion would be a good enhancement to clear up this potential accessibility compliance issue and I support the request.

**Comment by kathrynwp** [comment]
@nielslange Totally understand your POV as well.

In the past, we have not made visual A11y-related changes on request by one user – especially on a theme this old, and where the user has access to custom CSS – so this would set a precedent. 

Will let the theme wranglers decide whether to go ahead with this.

**Comment by kjellr** [comment]
I just chatted with @melchoyce and @allancole about this, and we all agree that while we should certainly be cautious about making color updates to older themes, this particular case seems safe to go ahead with. The existing color is _so close_ to being accessible. We don't expect the vast majority of users to notice once this is changed — it's hard to notice the difference ourselves when carefully comparing the screenshots. The upshot is that it'll make a lot of existing sites more accessible with a quick, simple change. 

Regarding the color though, @melchoyce suggested we use `#73757D` instead of `#767676`, as is currently substituted in #1769. This is even closer to the original color and also passes WCAG AA with a contrast ratio of 4.59.

**Comment by nielslange** [comment]
> Regarding the color though, @melchoyce suggested we use #73757D instead of #767676, as is currently substituted in #1769. This is even closer to the original color and also passes WCAG AA with a contrast ratio of 4.59.

@kjellr I've just changed the color from #767676 to #73757D as mentioned by you above.

**Comment by KokkieH** [comment]
Thanks, @mendezcode, and @nielslange for the fix. I'll let the user know.
